docs/prisma-schema-file.md

```diff
@@ -251,7 +251,23 @@ Tip: To quickly switch between environments you can `source` a file with the `ex
 export POSTGRES_URL=postgresql://test:test@localhost:5432/test?schema=public
 ```
 
-`$ source ./dev_env`
+Then run the following command:
+
+```bash
+source ./dev_env
+```
+
+### Using environment variables with Photon.js
+
+While the Prisma 2 CLI automatically picks up `.env` files, Photon.js doesn't natively support usage of [`dotenv`](https://github.com/motdotla/dotenv) or similar libraries that will do this. If you want to environment variables to be evaluated at runtime, you need to load them manually before instantiating `Photon` in your application code, e.g. using `dotenv`:
```

Comment:
This is quite a confusing sentence. 

Probably just shortening it to `... files, Photon.js does not natively support this. If you want to...` and then use the links to the library in there before the `:` will make it clearer.